src/main/scala/com/kinja/amqp/AmqpConfiguration.scala

```diff
@@ -26,6 +24,14 @@ final case class ResendLoopConfig(
 	memoryFlushTimeOut: FiniteDuration
 )
 
+object DeliveryGuarantee {
+	class Guarantee
+	case object AtLeastOnce extends Guarantee
+	case object AtMostOnce extends Guarantee
+}
```

Comment:
Which, for the record, is what `@enum` [expands to](https://github.com/ChrisNeveu/macrame/#getting-started).